tests/README.md

```diff
@@ -48,3 +48,60 @@ export TIFLASH_PORT=9000
 ./tiflash server -- --path /tmp/tiflash/data --tcp_port ${TIFLASH_PORT}
 storage_port=${TIFLASH_PORT} verbose=true ./run-test.sh delta-merge-test
 ```
+
+## How to run test cases in `fullstack-test-next-gen`
+
+For running next-gen integration test cases, you should define a TiDB cluster with disaggregated arch TiFlash node
+
+* 1 minio-server for providing object storage API
+* 1 PD
+* 1 next-gen TiKV
+* 1 TiDB
+* 1 TiFlash compute node
+* 1 TiFlash write node
+
+Notice: The pd, next-gen TiKV, tidb now is only available from pingcap internal dockerhub. Make sure you can access the dockerhub before running the tests.
+
+1. Build your own TiFlash binary using debug profile:
+
+    ```bash
+    # In the TiFlash repository root:
+    mkdir cmake-build-ng
+    cd cmake-build-ng
+    cmake -GNinja -DCMAKE_BUILD_TYPE=DEBUG -DENABLE_NEXT_GEN=ON ..
+    ninja tiflash -j32
+
+    # Copy the TiFlash binary to test directory
+    cmake --install . --component=tiflash-release --prefix "/path/to/your/tiflash/tests/.build/tiflash"
+    ```
+
+2. Run the tests using `docker compose`
+
+    ```bash
+    # In the TiFlash repository root:
+    cd tests/fullstack-test-next-gen
+
+    # Ensure `docker` or `podman` or `docker-compose` is installed in your env.
+    # This script will start a minimal next-gen cluster for running the tests.
+    # The next-gen TiKV, TiDB, PD, minio image will be pulled from pingcap internal
+    # dockerhub. Make sure you can access the dockerhub before running the tests.
+    ENABLE_NEXT_GEN=true ./run.sh
+
+    # If you want a more verbose running log
+    verbose=true ENABLE_NEXT_GEN=true ./run.sh
```

Comment:
If you still meet problem when running this test locally, try refer to the following gist:
https://gist.github.com/JaySon-Huang/c5fd4aaabbfed73d0bc216ff49e04ba2